tests/unit_tests/sql_parse_tests.py

```diff
@@ -1221,6 +1222,151 @@ def test_has_table_query(sql: str, expected: bool) -> None:
     assert has_table_query(statement) == expected
 
 
+@pytest.mark.parametrize(
```

Comment:
Reading through these tests I wonder how this handles aliases, something like
```
SELECT a.*, b.*
FROM tbl_a AS a
INNER JOIN tbl_b AS b ON a.col = b.col
```

also the shorthand form (or combo of them)

```
SELECT a.*, b.*
FROM tbl_a a
INNER JOIN tbl_b b ON a.col = b.col
```

Reply:
I added those two to the unit tests, it works as expected!

(I did have to update the mock `get_rls_for_table` to make it work, though!)